apps/client/src/features/app-settings/panel/project-panel/ProjectData.tsx

```diff
@@ -34,6 +38,36 @@ export default function ProjectData() {
     }
   }, [data, reset]);
 
+  const handleUploadProjectLogo = async (event: ChangeEvent<HTMLInputElement>) => {
```

Comment:
question: would it not be preferrable that the logo field is handled lile any other fields? ie: changes must be saved to take effect. so we dont call endpoints until the whole form is submitted